pkgs/kde/gear/falkon/default.nix

```diff
@@ -0,0 +1,14 @@
+{
+  mkKdeDerivation,
+  extra-cmake-modules,
+  qtwebchannel,
+  qtwebengine,
+  qttools,
+  python3Packages,
+}:
+mkKdeDerivation {
+  pname = "falkon";
+
+  extraNativeBuildInputs = [qttools qtwebchannel qtwebengine];
```

Comment:
Shouldn't webchannel and webengine be only in buildInputs?

Reply:
No, it has some native tools in there.